nautobot/docs/additional-features/graphql.md

```diff
@@ -127,4 +127,12 @@ Result
 }
 ```
 
+## Saved Queries
 
+Queries can now be stored inside of Nautobot, allowing the user to easily rerun previously defined queries.
+
+Inside of Extensibility > Data Management > GraphQL Queries, there are views to create and manage GraphQL queries.
```

Comment:
Looks like we're not self-consistent across all of our docs, but the Git datasource user guide uses the following:

```suggestion
Inside of **Extensibility -> Data Management -> GraphQL Queries**, there are views to create and manage GraphQL queries.
```